docroot/themes/custom/mass_theme/templates/content/node--topic-page.html.twig

```diff
@@ -67,20 +67,18 @@
           } %}
         {% endfor %}
       {% endblock %}
-      {# {% block secondaryRelationship %}
-        {% for item in secondary %}
-          {% include "@atoms/item-link.twig" with {
-            "itemLink": {
-              "title": item.title,
-              "href": item.href
-            }
-          } %}
-        {% endfor %}
-      {% endblock %} #}
     {% endembed %}
   {% endif %}
-
-  {% include "@organisms/page-banner/page-banner.twig" %}
+  {% include "@organisms/page-banner/page-banner.twig" with {
+    "pageBanner": {
+      "theme": "c-white",
```

Comment:
can get rid of the pageBanner wrapper and theme

Reply:
DOne